app/views/layouts/_navbar.html.erb

```diff
@@ -8,18 +7,14 @@
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
       </button>
-      <h4><%= link_to 'Logo', root_path %></h4>
+      <%= link_to image_tag("logo.jpg", width: '81', height: '57', :border=>0), root_path %>
```

Comment:
hi fer, the logo is still a WIP, we want to separate the words "Sushi Me Away" out of the Logo(bottom) and place it next to the cartoon logo. We removed the :border=>0 since it doesn't do anything anyway!